### PR TITLE
Handles binary data in cloud event.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Handles binary data in cloud event.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: https://github.com/dapr/dapr/issues/2612

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [X] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: contenttype/utils.go [hunk 1]
```diff
@@ -33,6 +33,11 @@ func IsStringContentType(contentType string) bool {
 	return isContentType(contentType, "application/xml")
 }
 
+// IsBinaryContentType determines if content type is byte[]
+func IsBinaryContentType(contentType string) bool {
+	return isContentType(contentType, "application/octet-stream")
```
Comment (jigargandhi) [inline contenttype/utils.go:38]: This is fine for now, later we will add proper content type detection

**Review by jigargandhi — APPROVED** [review]
Thank you for adding this

File: contenttype/utils.go (hunk 1 above)
Comment (artursouza) [reply contenttype/utils.go:38]: Correct. We should use a richer detection using MimeType hierarchy. This issue is about that: https://github.com/dapr/components-contrib/issues/635

**Review by yaron2 — APPROVED** [review]
lgtm

**Pull request merged**